two_factor/models.py

```diff
@@ -54,6 +54,11 @@ def key_validator(*args, **kwargs):
     """Wraps hex_validator generator, to keep makemigrations happy."""
     return hex_validator()(*args, **kwargs)
 
+def random_hex_str():
+    h = random_hex()
+    if isinstance(h, bytes):
```

Comment:
Reading the source of `random_hex`, it calls `binascii.hexlify` which returns `bytes` under Python 3 and `str` under Python 2 (remember that Python 2's `str` is Python 3's `bytes`)

I think this if statement could be removed.

Reply:
Oh, and flake8 isn't happy with your changes either:

```
two_factor/models.py:57:1: E302 expected 2 blank lines, found 1
two_factor/models.py:60:5: E101 indentation contains mixed spaces and tabs
two_factor/models.py:60:5: W191 indentation contains tabs
two_factor/models.py:63:1: E302 expected 2 blank lines, found 1
```

Reply:
I just wanted to be double sure that I don't brake any code, because in previous versions (also using Python 3) there wasn't such issue. So either random_hex function has changed or Python has changed in versions.

Reply:
ooh, that wasn't great idea to edit source in Github editor. :)